### DISCUSSION TITLE
v3.3.x

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4300] [Releases]
The **release manager** for `v3.3.x` is @stephenworsley

The **release buddy** for `v3.3.x` is @lbdreyer 

The release feature branch is [v3.3.x](https://github.com/SciTools/iris/tree/v3.3.x).

| Minor Release | Status | Milestone | Project | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.3.0rc0` | Completed | [v3.3.0](https://github.com/SciTools/iris/milestone/45) | [Iris v3.3.0](https://github.com/SciTools/iris/projects/33) | 17/08/2022 | [Whatsnew](https://scitools-iris.readthedocs.io/en/v3.3.0rc0/) | [v3.3.0rc0](https://github.com/SciTools/iris/releases/tag/v3.3.0rc0) | [Available](https://anaconda.org/conda-forge/iris?version=3.3.0rc0&channel=rc_iris) | [Available](https://pypi.org/project/scitools-iris/3.3.0rc0/) |
| `v3.3.0` | Completed | [v3.3.0](https://github.com/SciTools/iris/milestone/45) | [Iris v3.3.0](https://github.com/SciTools/iris/projects/33) | 01/09/2022 | [Whatsnew](https://scitools-iris.readthedocs.io/en/v3.3.0/whatsnew/3.3.html) | [v3.3.0](https://github.com/SciTools/iris/releases/tag/v3.3.0) | [Available](https://anaconda.org/conda-forge/iris?version=3.3.0&channel=main) | [Available](https://pypi.org/project/scitools-iris/3.3.0/) |

| Patch Release | Status | Milestone | Project | Date | RTD | Release | conda-forage | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.3.1` | Completed | - | - | 29/09/2022 | [Whatsnew](https://scitools-iris.readthedocs.io/en/v3.3.1/) | [v3.3.1](https://github.com/SciTools/iris/releases/tag/v3.3.1) | [Available](https://anaconda.org/conda-forge/iris?version=3.3.1&channel=main) | [Available](https://pypi.org/project/scitools-iris/3.3.1/) |

**Comment by bjlittle** [comment]
@stephenworsley Given the [release schedule of ESMValTool](https://docs.esmvaltool.org/en/latest/community/release_strategy.html), and given our current workload over the coming months, I propose that we target the release of `iris` 3.3.0 in **September**, which would then underpin  the `esmvaltool` 2.7.0 release in **October**.

This means that the `iris` 3.3.0rc0 release candidate would appear **~2-3 weeks** prior to **September**, so that's sometime in **August**.

Does that sound reasonable?

**Reply by bjlittle** [reply]
@bouweandela @zklaus @schlunma Would this timeline be an issue for you guys?

**Reply by zklaus** [reply]
That sounds good to me, thanks!

**Comment by bjlittle** [comment]
@SciTools/iris-devs Are we still happy using GH Milestones for releases? :thinking: 

We're managing release progress and status with GH Projects, which I find invaluable, but I'm on the fence about Milestones... I can see pros and cons either way.

Just thought I'd ask to see if anyone has an opinion about this, now that we've recently had several different release managers.

**Reply by wjbenfold** [reply]
I'm not sure I see much benefit of the Milestones over Projects?

**Comment by stephenworsley** [comment]
The release candidate is being delayed due to issues detailed in #4911. It's currently unclear why this hasn't fixed the docs build but we intend to release 3.3.0rc0 as soon as we are able to build the documentation.

**Reply by stephenworsley** [reply]
The release candidate is now available on pip but releasing on conda is stuck for the time being https://github.com/conda-forge/iris-feedstock/pull/83.

**Reply by stephenworsley** [reply]
The release candidate is now available on conda also.
